lib/ansible/modules/cloud/amazon/ec2_vpc_vgw.py

```diff
@@ -376,40 +369,33 @@ def ensure_vgw_present(client, module):
     params['Tags'] = module.params.get('tags')
     params['VpnGatewayIds'] = module.params.get('vpn_gateway_id')
 
-    # Check that a name argument has been supplied.
-    if not module.params.get('name'):
-        module.fail_json(msg='A name is required when a status of \'present\' is suppled')
+    # check that the vpc_id exists. If not, an exception is thrown
+    if params['VpcId']:
```

Comment:
just in case, can we do `params.get('VpcId')`?

Reply:
Wait, never mind. I think vpc ID is required for VGW's.

Reply:
It's not required but the .get() is done a few lines above: https://github.com/ansible/ansible/pull/38666/files#diff-c994632416321d31502ea68de645f97cL374